.github/workflows/process-load-test-result.yml

```diff
@@ -51,11 +51,11 @@ jobs:
       - name: Merge PR
         if: ${{ steps.processResults.outputs.hasPassed }}
         run: |
-          status="NOT COMPLETED"
-          while [ ! -z "$status" ]
+          checkCount="0"
```

Comment:
Shall we add the changes related to branch deletion as well
https://github.com/ballerina-platform/module-ballerina-ftp/pull/552/files

Reply:
Added in [723f80e](https://github.com/ballerina-platform/module-ballerinax-mysql/pull/380/commits/723f80e59d5f19519733b21222d1864585de30bf)